src/tests/MainTest.Java

```diff
@@ -108,6 +108,11 @@ class MainTest {
                     case 3:
                     case 4:
                     case 5:
+                        System.out.println("FORMAT: rowsort INBTNAME OUTBTNAME ROWORDER COLUMNNAME NUMBUF");
+                        System.out.println("ROWORDER: \n 1. Ascending\n 2. Descending" );
```

Comment:
Implement an external rowSort operator that results in a type 1 BigT in which the rows are sorted (in non decreasing order) according to the most recent values for the given column label

Rows are always supposed to be in ascending order. Refer the phase III doc.